ui/src/data/components/DataItem.jsx

```diff
@@ -51,15 +51,15 @@ function DataItem({ metadata, page }) {
         }
         rightActions={
           <>
-          {citationCount ? (
+          {citationCount != null && (
```

Comment:
lets make a strict check for consistency -> !==

Also, should we check against undefined too so as to be more precise ?

Reply:
Correct me if I'm wrong, wouldn't the immutable Map  always  give back `null` and not undefined?

Reply:
I think it is the other way around, no ? https://immutable-js.com/docs/v3.8.2/Map/#get()